ld/ldexp.h

```diff
@@ -66,6 +66,7 @@ typedef union etree_union {
     node_type type;
     const char *dst;
     union etree_union *src;
+    bool defsym;
```

Comment:
doesn't look microblaze related: to be inspected

Reply:
evert "ld: Remove unused expression state" --defsym symbol=expression Create a global symbol in the output file, containing the absolute address given by expression.

This reverts commit 65f14869fd3fbee8ed4c4ca49de8aaa86dbc66cb.

Reply:
Same as https://github.com/zephyrproject-rtos/binutils-gdb/pull/10#discussion_r1421607347